docker/Dockerfile

```diff
@@ -6,4 +6,5 @@ WORKDIR /home/synergify
 COPY . /home/synergify/
 
 RUN apt-get update && \
-    apt-get install htop gcc make -y
\ No newline at end of file
+    apt-get install htop gcc make -y && \
```

Comment:
it would be better to write it like this
apt-get install htop gcc make valgrind -y
as "-y" puts yes to any question that may be introduced during the installation